introduction.md

```diff
@@ -0,0 +1 @@
+I like programming in many different languages and codeschool is a fun way to learn some of these languages!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.